src/lib/server/index.ts

```diff
@@ -35,6 +37,10 @@ export async function createDb(connectionString: string): Promise<DB> {
     return drizzle(queryClient, { schema });
 }
 
+export const lower = (s: AnyPgColumn) => {
+    return sql`lower(${s})`;
+};
+
```

Comment:
I agree this is misplaced, but I also don't think the suggested location makes a ton of sense. For the sake of consistency, I have moved it to the suggested location anyways though.

Reply:
That location makes sense to me since they are utilities for the services.  I think the only change to the location would be to have it in a `sql.ts` file rather than the index.

Reply:
fair enough